src/seedsigner/models/settings.py

```diff
@@ -4,20 +4,15 @@
 from typing import Any, List
 
 from seedsigner.models.settings_definition import SettingsConstants, SettingsDefinition
-
 from .singleton import Singleton
 
 
 
 
 class Settings(Singleton):
-    hostname = os.uname()[1]
-    SETTINGS_FILENAME = ""
-
-    if hostname == "seedsigner-os":
-        SETTINGS_FILENAME = "/mnt/microsd/settings.json"
-    else:
-        SETTINGS_FILENAME = "settings.json"
+    HOSTNAME = os.uname()[1]
+    SEEDSIGNER_OS = "seedsigner-os"
```

Comment:
These are defined in `controller.py` also. Not sure which place makes the most sense. Controller?

Reply:
The reason for this is because the Settings singleton instance is instanciated prior to the Controller instance completing its instantiation. Settings needs to know at init the hostname to know where to look for the settings file on the microSD card. The Settings class currently has no knowledge of the Controller class (unlike the views/screens/gui).

I'll look to find a clean way to reduce the duplicate logic in the code base. Open to ideas.

Off the top of my head, I'm thinking I can pass the hostname into a Settings configure_instance method.